FusionIIIT/applications/counselling_cell/models.py

```diff
@@ -57,6 +57,9 @@ class CounsellingIssueCategory(models.Model):
     category_id = models.CharField(max_length=40)
     category = models.CharField(max_length=40)
 
+    class Meta:
+        unique_together = (('category_id', 'category'))
```

Comment:
This means that there can be any number of `category` for one `category_id` which is wrong.

Only `category_id` must be unique. Just add the `unique=True` argument in the `category_id` field.